### ISSUE TITLE
make it more difficult to change the settings

### ISSUE THREAD

**Issue by pavelstudeny** [ChrisSmith2/YouTube-Time#4]
Nice extension! When I had the idea about writing one, I realized someone must have written it already :-) Would it be within the goals of this extension to add a pseudo-protection of the settings e.g. by saving a password hash together with the settings?

**Comment by MoranLeven** [comment]
Yes its possible, rather than using a password try to use a strong passkey ( a one you can't memorize). And save it somewhere it is not easy to get you hand around. In dire situation you can get the passkey and slap it to overwrite the settings.
